lib/gpfpiso.gi

```diff
@@ -320,6 +320,7 @@ function(g,str,N)
       od;
 
       # we know sf is simple
+      IsSimpleGroup(sf);
```

Comment:
Hmm, what's the point of this? Doesn't this even slow down the code in some cases, because it might end up performing a simplicity check?
Also note that `IsNonabelianSimpleGroup` already implies `IsSimpleGroup`, so the next line would set the filter, too.

Reply:
Thanks. Indeed redundant now (and I think the result of merging older changes in, after other things happened. Will remove